homeassistant/components/jellyfin/config_flow.py

```diff
@@ -24,6 +25,12 @@
     }
 )
 
+REAUTH_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_PASSWORD, default=""): str,
```

Comment:
```suggestion
        vol.Required(CONF_PASSWORD): str,
```
I guess it's required to have a password which can't be ""?

Reply:
It is actually possible to remove a password from a user entirely and authenticate with just a username. Not very safe of course, but possible in Jellyfin.

Reply:
😱